presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/TypeConverter.java

```diff
@@ -261,6 +270,9 @@ private static TypeInfo toHiveTypeInfo(Type type)
         if (DATE.equals(type)) {
             return HIVE_DATE.getTypeInfo();
         }
+        if (type.equals(TIME_MICROS)) {
```

Comment:
Flip `equals()` so it's consistent with the others

Reply:
Flipped.